### PR TITLE
Rebuild with CB3 / GCC7;  update to 2.4

### PR DESCRIPTION
**Pull request by epruesse**

### CODE REVIEW

**Comment by epruesse** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,27 +13,15 @@ source:
...
-    - python           # [win]
-    - vc 9             # [win and py27]
-    - vc 10            # [win and py34]
-    - vc 14            # [win and py>=35]
-    - posix            # [win]
-    - pthreads-win32   # [win]
-    - m2w64-toolchain  # [win]
-    - toolchain
-    - zlib 1.2.*
+    - {{ compiler('c') }}
```
Comment (jakirkham) [inline recipe/meta.yaml:20]: Probably need this for Windows.

```suggestion
    - {{ compiler('c') }}               # [unix]
    - {{ compiler('m2w64_c') }}         # [win]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -13,27 +13,15 @@ source:
...
+
 
 requirements:
   build:
-    - python           # [win]
-    - vc 9             # [win and py27]
-    - vc 10            # [win and py34]
-    - vc 14            # [win and py>=35]
-    - posix            # [win]
-    - pthreads-win32   # [win]
```
Comment (jakirkham) [inline recipe/meta.yaml:28]: I think `posix` and `pthreads-win32` will still be needed for Windows.

Comment (jakirkham) [reply recipe/meta.yaml:28]: Though `pthreads-win32` may need to be listed in `host`.

**Comment by jakirkham** [comment]
Trying to fix this up in PR ( https://github.com/conda-forge/pigz-feedstock/pull/9 ). Though leaving the version update out for a subsequent PR.

**Pull request closed without merging**